pom.xml

```diff
@@ -13,8 +13,8 @@
 	<properties>
 		<java.version>1.7</java.version>
 		<kafka.version>0.10.1.1</kafka.version>
-		<spring-kafka.version>1.1.2.RELEASE</spring-kafka.version>
-		<spring-integration-kafka.version>2.1.0.RELEASE</spring-integration-kafka.version>
+		<spring-kafka.version>1.1.6.RELEASE</spring-kafka.version>
```

Comment:
Maybe in the future we can remove these entirely, as the baseline is now Boot 1.5